catenets/models/torch/metrics.py

```diff
@@ -0,0 +1,30 @@
+# third party
+import torch
+
+
+def sqrt_PEHE(y: torch.Tensor, hat_y: torch.Tensor) -> torch.Tensor:
```

Comment:
A few things here:
- ATE should be named abs_error_ATE or so (that's what the function computes)
- ATE and PEHE should be defined wrt the same inputs (here, hat_y seems to once be the treatment effect and once the two potential outcomes)
- I'd define both wrt the treatment effect, ie true_te and hat_te [or, if you really want to keep the dependence on two POs i'd call the first argument mu (so you have mu[:,1]-mu[:,0] instead of y[...))
- This file should probably be in the experiments folder, not here